tokenizers/src/pre_tokenizers/sequence.rs

```diff
@@ -6,19 +6,27 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, PartialEq)]
 #[macro_rules_attribute(impl_serde_type!)]
 pub struct Sequence {
-    pretokenizers: Vec<PreTokenizerWrapper>,
+    pre_tokenizers: Vec<PreTokenizerWrapper>,
```

Comment:
This seems like a breaking change to me.

Reply:
Though given this field is private I don't think it's a breaking change.

Reply:
I reverted it in either case.